cdap-docs/integrations/source/apache-sentry.rst

```diff
@@ -8,6 +8,34 @@
 Apache Sentry
 =============
 
+.. _apache-sentry-configuration:
+
+Configuring Apache Sentry for integration with CDAP
+---------------------------------------------------
+To use CDAP on a cluster using Apache Sentry for authorization, this property needs to be set:
+
+- Add the user ``cdap`` to the Sentry property ``sentry.service.allow.connect``
+
+We also, recommend setting these properties:
+
+- Add the user ``cdap`` to the Sentry property ``sentry.service.admin.group``; this allows
```

Comment:
Even without using the extension, we recommend adding allow.connect and admin.group. I am not completely sure why, but my best guess is for running explore on a Hive installation authorized by Sentry. These settings were present in documentation even before 3.5, I just moved them to the Sentry Integrations Manual. 

Also, the specific parts that the extension needs from these settings (sentry admin group) is mentioned in the sentry extension README already.